packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -1,42 +1,28 @@
-import React, { Component } from 'react';
-import classNames from 'classnames';
+import React, { PureComponent } from 'react';
 import { SelectableValue } from '@grafana/data';
-import { css } from 'emotion';
 import { Tooltip } from '../Tooltip/Tooltip';
-import { Icon } from '../Icon/Icon';
 import { ButtonSelect } from '../Forms/Legacy/Select/ButtonSelect';
-import memoizeOne from 'memoize-one';
-import { GrafanaTheme } from '@grafana/data';
-import { withTheme } from '../../themes';
+import { ButtonGroup, ButtonVariant, ToolbarButton } from '../Button';
 
 // Default intervals used in the refresh picker component
 export const defaultIntervals = ['5s', '10s', '30s', '1m', '5m', '15m', '30m', '1h', '2h', '1d'];
 
-const getStyles = memoizeOne((theme: GrafanaTheme) => {
-  return {
-    selectButton: css`
-      label: selectButton;
-      .select-button-value {
-        color: ${theme.palette.orange};
-      }
-    `,
-  };
-});
-
 export interface Props {
   intervals?: string[];
   onRefresh?: () => any;
   onIntervalChanged: (interval: string) => void;
   value?: string;
   tooltip?: string;
-  hasLiveOption?: boolean;
-  // You can supply your own refresh button element. In that case onRefresh and tooltip are ignored.
-  refreshButton?: React.ReactNode;
-  buttonSelectClassName?: string;
-  theme: GrafanaTheme;
+  isLoading?: boolean;
+  isLive?: boolean;
+  text?: string;
+  noIntervalPicker?: boolean;
+  'aria-label'?: string;
```

Comment:
We probably should standardise this as lots of the components define the prop as `ariaLabel`.

Reply:
hm... never seend that, we should use the html attribute name for it other wise it's going to be messy to refer to this property using two names

Reply:
Removed the need for this aria-label, moded this to a component aria-label that is defined in RefreshPicker